sentry_sdk/client.py

```diff
@@ -128,6 +133,11 @@ def _capture_envelope(envelope):
                     "auto_enabling_integrations"
                 ],
             )
+
+            sdk_name = _get_sdk_name(list(self.integrations.keys()))
+            SDK_INFO["name"] = sdk_name
```

Comment:
if we're mutating `SDK_INFO` in another module, it's not a `const` anymore, so I'd just move it to client.py.

Reply:
yes, you are right!